hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java

```diff
@@ -123,14 +123,24 @@ public BackendFeatures features() {
     public void open(HugeConfig config) {
         E.checkNotNull(config, "config");
 
+        if (this.sessions == null) {
+            synchronized(this) {
```

Comment:
add synchronized to open()
and don't set sessions volatile since we will check it with checkOpened() every time, same as cassandra